exalearn-api/ExaLearn.Bl/DTO/AudioQuestionDTO.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Shared.Enums;
+using System.Collections.Generic;
+
+namespace ExaLearn.Bl.DTO
+{
+    public class AudioQuestionDTO
+    {
+        public LevelType Level { get; set; }
+
+        public string FileId { get; set; }
+
+        public string Question { get; set; }
+
+        public List<AnswerDTO> AnswersId { get; set; }
```

Comment:
why are you using AnswersId? maybe change to AnswerId?

Reply:
you have list AnswerDTO, but not List<int>, it is not ids, rename to Answers